aziotctl/src/system.rs

```diff
@@ -38,12 +41,24 @@ pub struct LogLevelOptions {
     log_level: log::Level,
 }
 
-pub fn system(options: Options) -> Result<()> {
+#[derive(StructOpt)]
+#[structopt(about = "Reprovision device with IoT Hub")]
+pub struct ReprovisionOptions {
+    #[structopt(
+        value_name = "Identity Service URI",
+        long,
+        default_value = "unix:///var/run/aziot/identityd.sock"
+    )]
+    uri: url::Url,
```

Comment:
The default URL should be preceded by grabbing the URL from the config.toml file (in case it was overriden there). Right?

Reply:
We only allow overriding the default URL for debug builds, so it should be ok to always use "/run/aziot/identityd.sock". Anyways, this option is only available for debug builds too.

Reply:
Ah yes. Thanks for clarifying.

Reply:
Also the CLI shouldn't read the service's config file anyway.